src/network.c

```diff
@@ -2684,6 +2717,43 @@ static int network_config_set_interface(const oconfig_item_t *ci, /* {{{ */
   return 0;
 } /* }}} int network_config_set_interface */
 
+static int network_config_set_bind_address(const oconfig_item_t *ci,
+                                        struct sockaddr_storage **bind_address) {
+  char addr_text[256];
+
+  if (cf_util_get_string_buffer(ci, addr_text, sizeof(addr_text)) != 0)
+    return -1;
+
+  int ret;
+  struct addrinfo hint, *res = NULL;
+
+  memset(&hint, '\0', sizeof hint);
+  hint.ai_family = PF_UNSPEC;
+  hint.ai_flags = AI_NUMERICHOST;
+
+  ret = getaddrinfo(addr_text, NULL, &hint, &res);
+  if (ret) {
+    ERROR("Invalid address");
```

Comment:
This needs to be more descriptive and in one line.
Smth like to `ERROR('BindAddress option has invalid address set: %s', gai_strerror())`;